third_party/move/move-compiler-v2/tools/testdiff/src/main.rs

```diff
@@ -7,13 +7,16 @@
 
 use clap::Parser;
 use once_cell::sync::Lazy;
+use regex::{self, Regex};
 use std::{
     collections::{BTreeMap, BTreeSet},
     fmt::{Display, Formatter},
     path::PathBuf,
 };
 use walkdir::WalkDir;
 
+const DEBUG: bool = false;
```

Comment:
Maybe you can say here that the expectation is to set this manually to true, when trying to debug this tool?

Reply:
done